pkgs/development/python-modules/onnx-graphsurgeon/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, onnx
+, numpy
+}:
+
+buildPythonPackage {
+  pname = "onnx-graphsurgeon";
+  version = "0.3.25";
+
+  src = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "TensorRT";
+    rev = "8.5.2";
+    hash = "sha256-IvfMj1/m8NXEOHQdGTFMCy4ra1DuxFCEIDWvwymh7PU=";
+    sparseCheckout = [
+      "tools/onnx-graphsurgeon"
+    ];
+  };
+
+  preBuild = "cd tools/onnx-graphsurgeon";
+
```

There is currently a push to make this explicit, in hopes of removing this default in the future. :)
```suggestion

  nativeBuildInputs = [
    setuptools
  ];

```